release-notes/5.2/5.2.0-preview5.md

```diff
@@ -0,0 +1,106 @@
+# Release Notes
+
+## [Preview Release 5.2.0-preview5.24024.3] - 2024-01-24
+
+This update brings the below changes over the previous release:
+
+### Added
+
+- Added .NET 8 support [#2230](https://github.com/dotnet/SqlClient/pull/2230)
+- Added explicit version for major .NET version dependencies [#2303](https://github.com/dotnet/SqlClient/pull/2303)
```

Comment:
```suggestion
- Added explicit version for major .NET version dependencies on System.Runtime.Caching 8.0.0 and System.Configuration.ConfigurationManager 8.0.0 [#2303](https://github.com/dotnet/SqlClient/pull/2303)
```

Reply:
and System.Diagnostics.DiagnosticSource 8.0.0